README.md

```diff
@@ -408,6 +409,167 @@ options = StreamOptions(
+  from fauna.client import Client
+
+  client = Client()
+
+  response = client.query(fql('''
+  let set = Product.all()
+  {
+    initialPage: set.pageSize(10),
+    streamToken: set.toStream()
+  }
+  '''))
+
+  initialPage = response.data['initialPage']
+  streamToken = response.data['streamToken']
+
+  client.change_feed(streamToken)
+```
+
+You can also pass a query that produces a stream token directly to
+``change_feed()``:
+
+```python
+  query = fql('Product.all().changesOn(.price, .quantity)')
+
+  client.change_feed(query)
+```
+
+### Iterate on a Change Feed
+
+``change_feed()`` returns an iterator that emits pages of events. You can use a
+generator expression to iterate through the pages:
+
+```python
+query = fql('Product.all().changesOn(.price, .quantity)')
+
+with client.change_feed(query) as feed:
+  for page in feed:
+    print('Page stats: ', page.stats)
+
+    for event in page:
+      eventType = event['type']
+      if (eventType == 'add'):
+        print('Add event: ', event)
+        ## ...
+      elif (eventType == 'update'):
+        print('Update event: ', event)
+        ## ...
+      elif (eventType == 'remove'):
+        print('Remove event: ', event)
+        ## ...
+```
+
+Alternatively, you can iterate through events instead of pages with
+``flatten()``:
+
+```python
+query = fql('Product.all().changesOn(.price, .quantity)')
+
+with client.change_feed(query) as feed:
+  for event in feed.flatten():
+    eventType = event['type']
+    ## ...
+```
+
+The change feed iterator stops when there are no more events to poll.
+
+### Error handling
+
+If a non-retryable error occurs when opening or processing a change feed, Fauna
+raises a ``FaunaException``:
+
+```python
+from fauna import fql
+from fauna.client import Client
+from fauna.errors import FaunaException
+
+client = Client()
+
+try:
+  feed = client.change_feed(fql(
+    'Product.all().changesOn(.price, .quantity)'
+  ))
+  for event in feed.flatten():
+    print(event)
+    # ...
+except FaunaException as e:
+  print('error ocurred with change feed: ', e)
+```
+
+Errors can be raised at two different places:
+
+1. At the ``change_feed`` method call;
+2. At the page iteration.
+
+This distinction allows for users to ignore errors originating from event
+processing. For example:
+
+```python
+from fauna import fql
+from fauna.client import Client
+from fauna.errors import FaunaException
+
+client = Client()
+
+# Imagine if there are some products with details = null.
+# The ones without details will fail due to the toUpperCase call.
+feed = client.change_feed(fql(
+  'Product.all().map(.details.toUpperCase()).toStream()'
+))
+
+for page in feed:
+  try:
+    for event in page:
+      print(event)
+      # ...
+  except FaunaException as e:
+    # Pages will stop at the first error encountered.
+    # Therefore, its safe to handle an event failures
+    # and then pull more pages.
+    print('error ocurred with event processing: ', e)
+```
```

Comment:
@jrodewig @erickpintor This will need updated now along with the Erick's changes to error handling.

Reply:
Sounds good. I'll wait on guidance from @erickpintor. It looks like I also need to update the [broader Change Feeds ](https://github.com/fauna/docs.fauna.com/pull/2324)docs to cover error handling.